changelog.md

```diff
@@ -10,7 +10,15 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ### Changed
 - [Change default username for client initialization to `root`](https://github.com/ballerina-platform/ballerina-standard-library/issues/2397)
+
+## [1.4.1] - 2022-06-27
+
+### Added
+
+### Changed
+- [Fix incorrect code snippet in SQL api docs](https://github.com/ballerina-platform/ballerina-standard-library/issues/2931)
```

Comment:
```suggestion

```

ISnt this a duplicate?

Reply:
Fixed in [e2fc785](https://github.com/ballerina-platform/module-ballerinax-mysql/pull/536/commits/e2fc785b2157e0f5e2ed131c4d47665f14da010b)